### PR TITLE
Rename palette labels (add an s)

### PR DESCRIPTION
**Pull request by ivy-lli**
Before:
image

After:
image

### CODE REVIEW

**Review by ivy-rew — APPROVED** [review]

File: playwright/tests/page-objects/toolbar.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ export class Toolbar {
     await this.toolbar.getByRole('button', { name: 'Device mode' }).click();
   }
 
-  async openPalette(name: string) {
+  async openPalette(name: 'All Components' | 'Structures' | 'Elements' | 'Actions' | 'Composites') {
```
Comment (ivy-rew) [inline playwright/tests/page-objects/toolbar.ts:37]: I think 'All Components' a bulky term that blows up the palette. IMHO 'All' would be better ... but I think this is not the time to discuss it :wink:

Comment (ivy-lli) [reply playwright/tests/page-objects/toolbar.ts:37]: Would be fine for me. `All Components` is streamlined with the process editor where it is called `All Elements`.
I'm not sure about `Elements`, but we can change this later too...

Comment (ivy-rew) [reply playwright/tests/page-objects/toolbar.ts:37]: what do you think @ivy-bb ; shouldn't we abbreviate this bulky term? "All Components" ?

toolbar

Comment (ivy-bb) [reply playwright/tests/page-objects/toolbar.ts:37]: IMO All Components is fine. Like All Elements

**Pull request merged**